cryptographic_estimators/SDEstimator/SDAlgorithms/bjmm.py

```diff
@@ -30,29 +38,25 @@
 
 class BJMM(SDAlgorithm):
     def __init__(self, problem: SDProblem, **kwargs):
-        """
-        Complexity estimate of BJMM algorithm in depth 2,3
+        """Complexity estimate of BJMM algorithm in depth 2,3.
 
-        The algorithm was introduced in [BJMM12]_  as an extension of [MMT11]_.
+        The algorithm was introduced in [BJMM12] as an extension of [MMT11].
```

Comment:
weren't these underscores `[BJMM]_` important for the automatic generation of the citations?

Reply:
yes, I am not sure it was actually functional in the documentation generation but technically you need the underscores to generate the citations as links

Reply:
Ok, still is missing to reverse the [ABC]_ back to [ABC]

Reply:
yes maybe it is good to keep it, as for the later regex substitution a [xxx]_ might be more unique than [xxx]